deployments/cardano-prod.nix

```diff
@@ -6,7 +6,13 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["prod"];
+    services.dd-agent.tags = ["env:production"];
+
+    # DEVOPS-64: disable log bursting
+    services.journald.rateLimitBurst = 0;
```

Comment:
Hmm, AIUI, without this option, `journald` would trim an excessively chatty stream of similar messages to 1 summary message per 30s.

This change effectively disables that trimming, forcing all messages to appear in the log.

How would that help?